tests/tests-base/src/main/resources/benchmark/bigbluebutton/diagrams_2015/README.md

```diff
@@ -0,0 +1 @@
+[Source](https://github.com/bigbluebutton/bigbluebutton.github.io/blob/25582d381a27557d5e308f30a6e94a603b7438ba/_posts/2015-04-04-09architecture.md)
```

Comment:
Could we move the Benchmark stuff into a PR to Benchmark? The Benchmark is linked via git-subtree :)